### PR TITLE
Add support for TRENDnet switch

### PR DESCRIPTION
**Pull request by arrmo**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/trendnet.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+os: trendnet
+text: 'TRENDnet Switch'
+type: network
+icon: trendnet
+nobulk: true
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+      #mib_dir:
+      #    - trendnet
```
Comment (PipoCanaja) [inline includes/definitions/trendnet.yaml]: Please add the mib file (best) or remove this comment.

File: includes/polling/os/trendnet.inc.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2016 Neil Lathwood
+ * @author     Neil Lathwood <neil@lathwood.co.uk>
```
Comment (PipoCanaja) [inline includes/polling/os/trendnet.inc.php]: Please correct this copyright info.

File: includes/polling/os/trendnet.inc.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+$hardware = snmp_get($device, 'sysDescr.0', '-Osqnv');
```
Comment (PipoCanaja) [inline includes/polling/os/trendnet.inc.php]: sysdescr is already available as a variable, so it is not necessary to re-request this.

**Comment by Jellyfrog** [comment]
Also missing test data, please run `./scripts/save-test-data.php -o trendnet`

**Comment by arrmo** [comment]
> Also missing test data, please run `./scripts/save-test-data.php -o trendnet`

Yes, I missed that part, sorry! Added now - thanks!

File: includes/definitions/trendnet.yaml (hunk 1 above)
Comment (arrmo) [reply includes/definitions/trendnet.yaml]: Removed it, thanks!

File: includes/polling/os/trendnet.inc.php (hunk 2 above)
Comment (arrmo) [reply includes/polling/os/trendnet.inc.php]: Updated! Sorry, I thought this was all to be kept common - my mistake.

File: includes/polling/os/trendnet.inc.php (hunk 3 above)
Comment (arrmo) [reply includes/polling/os/trendnet.inc.php]: Found it! It's $device['sysDescr'] => using that now, thanks!

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-60-release-changelog-january-2020/10863/1

**Pull request merged**